### PR TITLE
Minor refactoring in the project

### PR DESCRIPTION
**Pull request by ayuhsya**
- Renamed `sample/` to `pymediawiki/`

- Created `helpers.py`: Has methods to fetch results of two different kinds.

- Created `constants.json`: Contains `property`:`prefix` key value pairs, to generate `continuetag` in `helpers.py`.

### CODE REVIEW

**Comment by abinashmeher999** [comment]
You are being modest by saying that this is minor :smile:. Looks great! Will review it tomorrow(technically today evening). Thanks for doing it.

File: pymediawiki/helpers.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    return ret
+
+def _append_results(currlist, newlist, prop, strip_chars, entry_prop):
+    ret = _strip_JSON(newlist, prop, strip_chars, entry_prop)
+    for key in ret:
+        if key not in currlist:
+            currlist[key] = []
+        currlist[key] += ret[key]
+
+def _fetch_results(url, payload, headers, prop, strip_chars, entry_prop):
```
Comment (abinashmeher999) [inline pymediawiki/helpers.py:27]: @ayuhsya I think all instances of `_fetch_results` can be replaced with `_fetch_dict_results`. The places where we are using the former should also return the correct result when the `WikiPage` is instantiated with multiple page ids.

**Review by abinashmeher999 — CHANGES_REQUESTED** [review]
Everything else looks good.

File: pymediawiki/helpers.py (hunk 1 above)
Comment (ayuhsya) [reply pymediawiki/helpers.py:27]: Okay, but then the response would be an array of `dict` objects and not an array of strings as it is, presently, using the `_fetch_results` method.

Comment (abinashmeher999) [reply pymediawiki/helpers.py:27]: Yeah, no problem. Let us go with the generic approach (the one returning dictionary) for now. The interface for single and multiple pages will need a little more thought.

**Review by abinashmeher999 — APPROVED** [review]

**Pull request merged**